background.ts

```diff
@@ -517,6 +520,10 @@ ipcMainProxy.handle('settings-write', (event, arg) => {
 
 mainEvents.on('settings-write', writeSettings);
 
+ipcMainProxy.on('extensions-list', () => {
+  new BackgroundCommandWorker().listExtensionsMetadata();
```

Comment:
I'm not too sure about instantiating a new `BackgroundCommandWorker` for each time we receive a message for `extensions-list`. Perhaps it would make more sense to scope the `commandWorker` similarly to `httpCommandServer`.

Reply:
That could work, or just extract the list extension functionality to a function (not attached to `BackgroundCommandWorker`)?

Reply:
I don't know.. moving function `listExtensionsMetadata()` outside of the background command worker feels kinda messy because we depend on the extension manager, which is imported in the `BackgroundCommandWorker`. It almost feels like we can extract extension behavior out into an "ExtensionCommandWorker" of sorts.

Reply:
I moved this function out of the background command worker. Let me know what you think 🙂